client/libs/theme/_fonts.scss

```diff
@@ -17,8 +17,11 @@
 
 // Text Color
 .primary-text { color: $primary !important; }
+.bg-primary { background-color: $primary !important; }
 .accent-text { color: $accent !important; }
+.accent-icon { color: $accent !important; }
```

Comment:
Wherever you used `accent-icon` replace that with `accent-text` since the material icons are technically text.
```suggestion
```